travis_build_test.sh

```diff
@@ -11,15 +11,11 @@ start eslint
 ./node_modules/eslint/bin/eslint.js karma.conf.js app test
 end eslint
 
-start examples
-REGEN_SCRIPT='./docs/examples/regenerate-index.py'
-INDEX_JSON='docs/examples/index.json'
-diff --ignore-space-change <( $REGEN_SCRIPT ) $INDEX_JSON \
-  || die "Update fixture index: '$REGEN_SCRIPT > $INDEX_JSON'"
-# Compare the output of the regen script to the current index...
-# If there's a difference, it will be shown,
-# along with a one-liner to fix the problem.
-end examples
+start viewconfs
+CMD='./docs/examples/regenerate-index.py docs/examples/viewconfs test/view-configs test/view-configs-more'
```

Comment:
Comment please? What is this doing and why?

Reply:
It was regenerating lists of the files in each of these directories, so the JS tests could iterate over all.... but it's simpler just to get rid of the JS tests which can't list a directory.